test/api/UsersListFilteringSpec.scala

```diff
@@ -0,0 +1,120 @@
+package api
+
+import grid.FilterColumns
+import org.scalatest.{BeforeAndAfter, FunSpec}
+import org.scalatest.mock.MockitoSugar
+
+import org.scalatestplus.play._
+
+import play.api.mvc._
+import scala.concurrent.duration._
+import scala.concurrent.Await
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class UsersListFilteringSpec extends FunSpec with Results with MockitoSugar with OneAppPerSuite with BeforeAndAfter with TestUtils{
+  before{
+    val users = for {
+      _ <- clearDb
+      _ <- createUser("Frank@javeo.eu", "Frank", "Kowalski", 1, 1, 2, "notes1")
+      _ <- createUser("Frank2@javeo.eu", "Frank2", "Kowalski2", 2, 111, 2, "notes2")
+      _ <- createUser("Frank3@javeo.eu", "Frank3", "Kowalski3", 3, 112, 2, "notes3")
+    } yield ()
+
+    Await.result(users, 10 seconds)
+  }
+
+  describe("Filtering") {
+    it("filters properly with email param"){
+      for{
+        allUsers   <- usersListRequest(1, numberOfAllUsers, "", "", FilterColumns(Map("user.email" -> "Frank")))
```

Comment:
many "", "" -> add default for **sortBy** and **sortDir** within userListRequest method